hack/pin-bundle-images.sh

```diff
@@ -86,5 +86,6 @@ for MOD_PATH in $(go list -mod=readonly -m -json all | jq -r '. | select(.Path |
 done
 # append the rabbitmq URL only if we aren't in Dockerfile mode
 if [ -z "$DOCKERFILE" ]; then
-    echo -n ",quay.io/openstack-k8s-operators/rabbitmq-cluster-operator-bundle@sha256:2a09973faa91ae6d3913d61cc0ae7e124240c27f886563de87a2fbdd3aec6ad0"
+    # pin rabbit to sha256 for the e2d413c102bc73d4b5e186d1d1b1f9bf728701e1 tag (v2.6.0)
+    echo -n ",pull quay.io/openstack-k8s-operators/rabbitmq-cluster-operator-bundle@sha256:b3e37b529e76faf5b140021b3e5fd9ba5ef8cfa7c26e74845cf47d075e204906"
```

Comment:
Should this `pull` be here?  It seems to be showing up in the CI failures as if it were the actual name of an image:

```
time="2024-02-19T18:43:12Z" level=fatal msg="initializing source docker://pull:latest: reading manifest latest in docker.io/library/pull: requested access to the resource is denied"
```

Reply:
ugh, copy paste error on my part from quay.io